src/services/upload/exifService.ts

```diff
@@ -0,0 +1,100 @@
+import EXIF from 'exif-js';
+
+import { logError } from 'utils/sentry';
+import { NULL_LOCATION, Location } from './metadataService';
+
+const SOUTH_DIRECTION = 'S';
+const WEST_DIRECTION = 'W';
+const EXIF_HAVING_TYPES = new Set(['jpeg', 'jpg', 'tiff']);
```

Comment:
How do we handle exif data for HEIC file types? 
Also, is this list based on our observation or it's standard list? (Want to check that we are not missing out any other common types here)